src/ui/components/Toolbar.tsx

```diff
@@ -128,6 +128,25 @@ function InfoIcon() {
   );
 }
 
+function ProtocolIcon() {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M8.00033 7L1.70716 13.2929C1.31661 13.6834 1.31661 14.3166 1.70716 14.7071L8.00033 21"
+        stroke="currentColor"
+        strokeWidth="2"
+        strokeLinecap="round"
+      />
+      <path
+        d="M16 21L22.2929 14.7071C22.6834 14.3166 22.6834 13.6834 22.2929 13.2929L16 7"
+        stroke="currentColor"
+        strokeWidth="2"
+        strokeLinecap="round"
+      />
+    </svg>
+  );
+}
+
```

Comment:
This is an improvement over what was there before:
<img width="70" alt="Screen Shot 2023-07-28 at 5 34 09 PM" src="https://github.com/replayio/devtools/assets/29597/9200397e-4c1d-42d1-a727-8405abd5b695">

But it's still misaligned:
<img width="52" alt="Screen Shot 2023-07-31 at 9 08 02 AM" src="https://github.com/replayio/devtools/assets/29597/4ca53269-2e2c-4fd0-b5dc-fe01ee003669">

Reply:
This little guy is pretty off too
<img width="49" alt="Screen Shot 2023-07-31 at 9 12 26 AM" src="https://github.com/replayio/devtools/assets/29597/45a6d8d6-fad0-466d-8ea7-77e6d27585c5">

Reply:
I'll take a pass at this, this morning.

Reply:
975853da7